### DISCUSSION TITLE
osu!lazer being stuck importing beatmaps from stable (database issue probably)

### DISCUSSION THREAD

**Discussion by KitkatOsu** [ppy/osu#16229] [Q&A] [answered]
I wanted to import my stable beatmaps to lazer, it was working for a time and then at some point it got stuck and is still in that state.
I suppose its a database issue but I could be wrong.
Errors started at 2021-12-23 15:22:20 in the log file.
Any help? 
(Database log attached)
[database.log](https://github.com/ppy/osu/files/7770174/database.log)

**Comment by peppy** [comment, marked as answer]
Hard to say what went wrong, but looks to be threads stuck in a bad state. You can resume the process by restarting lazer and then running the import step again from settings. It will resume from where it left off.
